src/pudl/analysis/service_territory.py

```diff
@@ -450,15 +449,14 @@ def dagster_compile_geoms(
             save_format=save_format,
         )
 
-    return [dagster_compile_geoms]
```

Comment:
Is there a reason this was being stored in a list before and isn't any more?

Reply:
I also found this confusing, but it seemed unnecessary, and it resulted in a second layer of iteration when generating the assets which also seemed unnecessary, and I removed it kind of expecting to break something, but then nothing broke, so it really did seem to be unnecessary. 🤷🏼